CI 这边 **Android cargo check 挂了**，阻塞合并。Windows / macOS / Linux 都过了。

**失败点**
```
error[E0422]: cannot find struct, variant or union type `LocalAsrEngineStatus` in module `crate::commands`
  --> src/coordinator.rs:3240:35
```

**根因（看起来是）**
新增的 `emit_local_asr_engine_status` 在 `coordinator.rs` 里直接构造了 `crate::commands::LocalAsrEngineStatus`，但 `commands/mod.rs` 里 `local_asr` 模块是 `#[cfg(not(mobile))]` 导出的——Android 目标下 `crate::commands` 里没有这个类型。coordinator 侧的 `local_asr_cache` / `schedule_local_asr_release` 等则在全平台都会编译，所以 Android CI 会在这里炸。

**修复方向（任选其一即可）**
1. 把 emit payload 的类型挪到 mobile 也会编译的模块（例如 `types.rs`），coordinator 和 commands 共用；
2. 在 coordinator 里定义一个私有小 struct（字段同 `LocalAsrEngineStatus`），emit 时用 serde 对齐前端即可；
3. 给 `emit_local_asr_engine_status` 及其调用点加 `#[cfg(not(mobile))]`（Android 本来也没有 LocalAsr 页面，事件驱动可以只在 desktop 启用）。

修完后麻烦 push 一下，我这边会再看 Android cargo check 是否变绿。Thanks!

--- a/openless-all/app/src-tauri/src/coordinator.rs
+++ b/openless-all/app/src-tauri/src/coordinator.rs
@@ -3233,7 +3233,9 @@ fn ensure_local_qwen3_model_ready() -> Result<(), String> {
 /// 引擎加载/释放/keepLoadedSecs 变化时主动推给前端，前端 listen
 /// `local-asr:engine-changed` 即可零轮询同步 UI（issue #470 / #6）。
 /// 只反映 Qwen3 这一路（loaded_model_id / prefs），不碰 Foundry / Sherpa。
-/// 仅用跨平台符号，保证 Windows / Linux 也能编译。
+/// 仅用桌面端跨平台符号；Android 无本地 ASR 引擎（LocalAsrEngineStatus 不在该 target
+/// 编译），单独给 no-op stub（见下），让各调用点在所有平台统一编译。
+#[cfg(not(target_os = "android"))]
 fn emit_local_asr_engine_status(inner: &Arc<Inner>) {
     let model_id = inner.local_asr_cache.loaded_model_id();
     let keep_loaded_secs = inner.prefs.get().local_asr_keep_loaded_secs;
@@ -3246,6 +3248,10 @@ fn emit_local_asr_engine_status(inner: &Arc<Inner>) {
         let _ = app.emit("local-asr:engine-changed", &status);
     }
 }
+
+/// Android no-op：该 target 不编译 LocalAsrEngineStatus / 本地 ASR 引擎。issue #470 / #6。
+#[cfg(target_os = "android")]
+fn emit_local_asr_engine_status(_inner: &Arc<Inner>) {}
 
 /// 一次 dictation 结束后，按 prefs.local_asr_keep_loaded_secs 决定何时释放
 /// 内存里的 Qwen3-ASR 引擎。0 = 立即释放；其它值 = sleep N 秒后看 last_used。

